instance/static/html/instance/appserver.html

```diff
@@ -0,0 +1,121 @@
+<h4>{{appserver.name}}</h4>
+
+
+<div ng-if="!appserver">
+  <p>Loading AppServer details...</p>
+</div>
+
+<accordion close-others="false" ng-if="appserver">
+
+  <accordion-group heading="Status" is-open="true">
+    <table>
+      <tbody>
+        <tr>
+          <th>Status</th>
+          <td><strong>{{appserver.status_name}}</strong> ({{appserver.status_description}})</td>
+        </tr>
+        <tr>
+          <th>Active</th>
+          <td><span ng-if="is_active">Yes</span><span ng-if="!is_active">No</span></td>
+        </tr>
+        <tr>
+          <th>Created</th>
+          <td>{{appserver.created | date:'yyyy-MM-dd HH:mm:ssZ' }}</td>
```

Comment:
@bradenmacdonald Inconsistent spacing after/before opening/closing braces. Let's default to adding a single space since you've already updated other files to default to that. There's other lines above and below this one that have consistent spacing but will need to be updated to include a space.